ironfish/src/wallet/wallet.test.slow.ts

```diff
@@ -4,6 +4,7 @@
 import { Asset, generateKey } from '@ironfish/rust-nodejs'
 import { Assert } from '../assert'
 import { Target } from '../primitives/target'
+import { Account } from './account'
```

Comment:
In line 654 of this file, I get that
`    node.wallet.accounts.set(viewOnlyAccount.id, viewOnlyAccount)`
is trying to access inaccessible member `accounts`. Did I do a bad merge @jowparks?

Reply:
Hmm not sure, lets pair and we can post results back in PR here

Reply:
This ended up being a temporary issue while the main PR gets tested. Going to ignore.